app/src/main/java/org/simple/clinic/home/HomeScreen.kt

```diff
@@ -89,4 +93,20 @@ class HomeScreen(context: Context, attrs: AttributeSet) : RelativeLayout(context
   fun openFacilitySelection() {
     screenRouter.push(FacilityChangeScreenKey())
   }
+
+  fun showHelpButton() {
+    showHelpMenuItem
+        .apply {
+          isEnabled = true
+          isVisible = true
+        }
+  }
+
+  fun hideHelpButton() {
```

Comment:
Can we combine these two show/hide functions by passing in a boolean, just the way we've done at `HelpScreen#showWebview`?

Reply:
Sure.